public/documents/fari-rpgs/charge-rpg.md

```diff
@@ -475,31 +475,31 @@ The action rating is used to define the initial dice pool of the roll.
-- In a `risky` position, the situation is dangerous. If you fail, there will be repercussions.
-- In a `desperate` position, the odds are against you. Attempting to go against them might backfire heavily.
+- When the risk is `low`, things are under control. Even if things go wrong, there is a way out.
+- When the risk is `moderate`, the situation is dangerous. If you fail, there will be repercussions.
+- When the risk is `high`, the odds are against you. Attempting to go against them might backfire heavily.
 
-By default, if you roll to overcome something, you are probably in a `risky` position. You wouldn’t be rolling if there was no risk involved.
+By default, an action roll has a `moderate` risk. You wouldn’t be rolling if there was no risk involved.
 
-The GM uses the narrative context to determine which position to pick.
+The GM uses the narrative context to determine which risk to pick.
 
-Another thing that might affect the position is the `action` being used. If a merchant you are trying to convince is in a bad mood, using `Sway` might be pretty `desperate`. `Bond` on the other hand could put you in a more `controlled` position.
+The action being used is another thing that might affect the risk of a roll — you are trying to convince a merchant who's in a bad mood, using `Sway` might be pretty `high risk`. `Bond` on the other hand could put you in a more `low risk` position, considering the situation.
 
-The position helps determine what failure looks like.
+The risk helps determine what failure looks like.
 
 ## 3. GM Sets the Effect Rating
 
-The position isn't the only thing that helps resolve the end result of a roll. Even in a desperate position, succeeding on a roll might still have a great `effect` on the narrative.
+The risk isn't the only thing that helps resolve the end result of a roll. Even in a high risk situation, succeeding on a roll might still have a great `effect` on the narrative.
 
 In Charge, there are 3 possible effect ratings: `great`, `standard`, and `limited`. In most common scenarios, if you are rolling to overcome something, you are probably looking at generating a `standard` effect. The GM is responsible for telling you this information.
 
-If a heavy door is blocking your way, you could use `Finesse` to pick the lock. This might generate a `standard effect`, since it will take some time to do, but it's a more `controlled position`. You could also try to use `Tinker` and wreck the door open with an explosive. This may create a `great effect`, but might be a pretty `desperate` position, since it could create a lot of unwanted attention.
+If a heavy door is blocking your way, you could use `Finesse` to pick the lock. This might generate a `standard effect`, since it will take some time to do, but it's the `risk is low`. You could also try to use `Tinker` and wreck the door open with an explosive. This may create a `great effect`, but might be a `high risk` situation as it could create a lot of unwanted attention.
```

Comment:
I'd reword this one a bit:

```suggestion
If a heavy door is blocking your way, you could use `Finesse` to pick the lock. This might generate a `standard effect`, since it will take some time to do, but the roll would be `low risk`. You could also try to use `Tinker` and wreck the door open with an explosive. This may create a `great effect`, but might be a `high risk` situation as it could create a lot of unwanted attention.
```